pkg/diff/diff.go

```diff
@@ -59,14 +60,14 @@ func WithCompareNumberAndNumericString() Option {
 }
 
 // WithIgnoredPaths configures ignored fields.
-func WithIgnoredPaths(paths []string) Option {
+func WithIgnoreConfig(config map[string][]string) Option {
```

Comment:
change the comment on above this function header, the name is not matched

Reply:
:+1: